pages/human-donors/[uuid].js

```diff
@@ -125,6 +100,17 @@ export async function getServerSideProps({ params, req }) {
           filterErrors: true,
         })
       : [];
+    const phenotypicFeatures = donor.phenotypic_features
+      ? await request.getMultipleObjects(donor.phenotypic_features, null, {
+          filterErrors: true,
+        })
+      : [];
+    const phenotypeTermsList = phenotypicFeatures
```

Comment:
`phenotypicFeatures` always evaluates truthy because it gets assigned to the empty array if it couldn’t get anything from the server or didn’t exist in the first place. You can test `phenotypicFeatures.length > 0` here.